.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Leave a blank line at the end of the file.

Reply:
Resolved